packages/davinci-client/src/lib/collector.utils.ts

```diff
@@ -583,28 +613,32 @@ export function returnReadOnlyCollector(field: ReadOnlyField, idx: number) {
 
 /**
  * @function returnValidator - Creates a validator function based on the provided collector
- * @param collector {ValidatedTextCollector} - The collector to which the value will be validated
+ * @param {ValidatedTextCollector | ObjectValueCollectors | MultiValueCollectors} collector - The collector to which the value will be validated
  * @returns {function} - A "validator" function that validates the input value
  */
-export function returnValidator(collector: ValidatedTextCollector) {
+export function returnValidator(
```

Comment:
Are we providing a "validator" method for this phone number collector? If so, what does it actually do?